src/styles/sass/img/contents.svg

```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 24 24">
```

Comment:
Ah, no, those are for reference only, they are not used. I put them there for ease of transport from one computer to the next. We can remove them from the project for tidiness.